### ISSUE TITLE
Doesn't Run on Vista 64 bit

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MziRintu/iphonebrowser#49]
```
What steps will reproduce the problem?
1. Install on Vista 64 bit
2. Run Application
3. It crashes

What is the expected output? What do you see instead?
That the application will launch and work

What version of the product are you using? On what operating system?
1.52 from the install file: SetupiPhoneBrowser.1.52

Please provide any additional information below.
Tried in various compatibility modes none worked.

Here is the details from the crash dialog:

Description:
  Stopped working

Problem signature:
  Problem Event Name:   CLR20r3
  Problem Signature 01: iphonebrowser.exe
  Problem Signature 02: 1.5.2.0
  Problem Signature 03: 478c3bc0
  Problem Signature 04: iPhoneBrowser
  Problem Signature 05: 1.5.2.0
  Problem Signature 06: 478c3bc0
  Problem Signature 07: f
  Problem Signature 08: c6
  Problem Signature 09: System.InvalidOperationException
  OS Version:   6.0.6001.2.1.0.256.1
  Locale ID:    1033


```

Original issue reported on code.google.com by `rsfo...@gmail.com` on 25 Aug 2008 at 5:58

**Comment by GoogleCodeExporter** [comment]
```
same for me on Vista 64
```

Original comment by `flatz.ma...@gmail.com` on 31 Aug 2008 at 6:21

**Comment by GoogleCodeExporter** [comment]
```
Fixed in iPhoneBrowser v1.70
```

Original comment by `NetMage....@gmail.com` on 11 Sep 2008 at 6:05
- Changed state: **Fixed**
